src/libs/antares/timeelapsed/csv_writer.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <yuni/io/file.h>
+
+#include "i_writer.h"
+#include "content_handler.h"
+
+namespace TimeElapsed
+{
+class CSVWriter : public IWriter
+{
+public:
+    CSVWriter() = default;
+    void initialize(const Yuni::String& filename, ContentHandler* handler);
+    void flush() override;
+
+private:
+    Yuni::IO::File::Stream mOutputFile;
```

Comment:
Why do you leave **mOutputFile** and **contentHandler_** in the **CSVWriter** ?
Should they be in the **IWriter** (if you want a json writer, you'll inherit them from the **IWriter**)

Reply:
Ok, but if we have to develop a **jsonWriter**, what do we do ? 
Would we have to duplicate these data member from **CSVWriter** ?